examples/CSharp/WebGPU/WebGPUTexturedQuad/Program.cs

```diff
@@ -0,0 +1,650 @@
+using System.Numerics;
```

Comment:
There is not enough explanation/walkthrough here for this to be an example. Moreover, it is not structured like an example i.e. it should be `examples/CSharp/WebGPU Tutorials/Tutorial X.Y - Name`. 

We do have a rough tutorial structure planned for modern graphics APIs:
```
examples
    CSharp
        Vulkan Tutorials
            Tutorial 1.1 - Vulkan Init & Device
            Tutorial 1.2 - Extensions
            Tutorial 1.3 - Hello Window
            Tutorial 1.4 - Swapchain
            Tutorial 1.5 - Pipelines
            Tutorial 1.6 - Command Buffers
            Tutorial 1.7 - Shaders
            Tutorial 1.8 - Hello Quad
            Tutorial 2.1 - whatever you do after you've got a quad
        Vulkan Demos
            Compute Demo
```
(from #team-talk pins)

Obviously WebGPU will be slightly different but it should roughly follow that structure to ensure ample understanding between tutorials.

This probably doesn't qualify as a demo either given there is no explanation in here.

Recommend moving to the Lab.